ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/tests/TransactionService.scala

```diff
@@ -670,6 +671,23 @@ class TransactionService(session: LedgerSession) extends LedgerTestSuite(session
       }
     }
 
+  private[this] val transactionStreamingOfManyTransactions =
+    LedgerTest("TXStreamingOfManyTransactions", "Loading many transactions properly works.") {
```

Comment:
This is leaking knowledge of the implementation into the integration tests. I'm thinking of removing the test, since the more targeted test for the JDBC implementation is testing that the paging and efficient loading of transactions is working as expected.